assets/browser-fetch.js.t

```diff
@@ -2,15 +2,30 @@
   define('fetch', ['ember', 'exports'], function(Ember, self) {
     'use strict';
     var Promise = Ember['default'].RSVP.Promise;
-    if (global.FormData) {
-      self.FormData = global.FormData;
-    }
-    if (global.FileReader) {
-      self.FileReader = global.FileReader;
-    }
-    if (global.Blob) {
-      self.Blob = global.Blob;
+
+    var supportProps = [
+      'FormData',
+      'FileReader',
+      'Blob',
+      'URLSearchParams',
+      'Symbol',
+      'iterator',
+      'ArrayBuffer'
+    ];
```

Comment:
I think if we are to add all of these we should also add `fastboot` polyfills in order to ensure parity.

Reply:
Why are we adding `FormData`, `FileReader`, `Blob` to self? They are also not added to Fastboot fetch.

Reply:
Just walked through PR list, these have been merged and are exported, but not in Fastboot...
#108 #93 #76